tests/unittest/llmapi/test_llm_pytorch.py

```diff
@@ -428,24 +428,27 @@ def test_llama_7b_peft_cache_config_affects_peft_cache_size():
 
     # Test that too small PeftCacheConfig.host_cache_size causes failure
     with pytest.raises(RuntimeError):
-        check_llama_7b_multi_lora_from_request_test_harness(
-            LLM,
-            lora_config=lora_config_no_cache_size_values,
-            peft_cache_config=PeftCacheConfig(
-                host_cache_size=1),  # size in bytes
-            # Disable CUDA graph
-            # TODO: remove this once we have a proper fix for CUDA graph in LoRA
-            cuda_graph_config=None)
+        with try_expose_error_in_ray(RuntimeError):
```

Comment:
Combine the 2 ctx mgrs to reduce nesting: https://docs.python.org/3/reference/compound_stmts.html#the-with-statement:~:text=With%20more%20than%20one%20item%2C%20the%20context%20managers%20are%20processed%20as%20if%20multiple%20with%20statements%20were%20nested%3A